src/core/dev_api/openvino/core/type/float_util.hpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cstdint>
+#include <cstring>
+
+namespace ov {
+
+constexpr uint32_t three_bytes_shift = 24;
+
+namespace util {
+namespace {
+
+/**
+ * @brief Reinterpret float value to 32 bits.
+ *
+ * @param value  Input float value.
+ * @return       The 32-bite value
+ */
+inline uint32_t f32_to_u32_bits(float value) {
+    uint32_t bits;
+    static_assert(sizeof(bits) == sizeof(value), "'bits' must have same number bits as `float`");
```

Comment:
Is it possible otherwise?

Reply:
Decide to remove asserts, as there is no much benefit